src/main/java/com/icosillion/podengine/models/ITunesInfo.java

```diff
@@ -10,7 +10,7 @@
 public abstract class ITunesInfo {
 
     public enum ExplicitLevel {
-        EXPLICIT, CLEAN, UNKNOWN
+        EXPLICIT, CLEAN, NO, UNKNOWN
     }
 
     protected final Element parent;
```

Comment:
Your changes to this file in this commit (and perhaps other changes?) don't appear in the final diff of the PR. I wonder they were lost in the conflict resolution step? This is why I hate merges.